### PR TITLE
New KMS Example

### PR DESCRIPTION
**Pull request by bisdavid**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: awscli/examples/kms/get-key-policy.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+**To copy a key policy from one CMK to another CMK**
+
+The following ``get-key-policy`` example gets the key policy from one CMK and saves it in a text file. Then, it replaces the policy of a different CMK using the text file as the policy input.
+Because the ``--policy`` parameter of ``put-key-policy`` requires a string, you must use the ``--output text`` option to return the output as a simple text string.
```
Comment (juneb) [inline awscli/examples/kms/get-key-policy.rst]: Please delete "simple text". It's just a string.

Comment (bisdavid) [reply awscli/examples/kms/get-key-policy.rst]: My intent was to distinguish between **--output text** and **--output json**, which is also, technically a string.

Comment (bisdavid) [reply awscli/examples/kms/get-key-policy.rst]: I'll change it to "...return the output as a simple string instead of JSON."

Comment (juneb) [reply awscli/examples/kms/get-key-policy.rst]: How about "return the output as a text string instead of JSON."

**Comment by supertat55** [comment]
unscribe

2019年7月12日(金) 7:48 June Blender <notifications@github.com>:

File: awscli/examples/kms/get-key-policy.rst (hunk 1 above)
Comment (bisdavid) [reply awscli/examples/kms/get-key-policy.rst]: Ok.  Will do.

Comment (bisdavid) [reply awscli/examples/kms/get-key-policy.rst]: I changed it as you requested - however, JSON is also a text string, so I'm not sure that's the best way to explain it.

**Comment by bisdavid** [comment]
This should be good now.  Please merge.

**Pull request merged**